crowbar_framework/lib/crowbar/upgrade_timeouts.rb

```diff
@@ -0,0 +1,37 @@
+require "yaml"
+
+module Crowbar
+  class UpgradeTimeouts
+    def values
+      @timeouts_config = begin
+        YAML.load_file("/etc/crowbar/upgrade_timeouts.yml")
```

Comment:
Hm, so this is reloading the yaml file for every ```values``` call. I guess it's not a bit deal as there are not so many calls. But it's not really nice either.